core/src/ingestion.ts

```diff
@@ -6,6 +6,7 @@ import { Change } from "./entities/Change"
 import { FetchedRecord } from './fetched-record'
 import { FetchedRecordBuffer } from './fetched-record-buffer'
 import { stitchFetchedRecords } from './stitching'
+import { Consumer } from 'nats';
```

Comment:
Nit: can we move this import closer to `@mikro-orm/postgresql` to separate it from local imports? (this can probably be enforced automatically with linters later)